internal/config/config.go

```diff
@@ -44,6 +44,7 @@ const (
 	GATEWAY_ACTIVE_CONNECTION_REGISTRAR_POLL_MAX_DELAY = "Gateway_Active_Connection_Registrar_Poll_Max_Delay"
 	GATEWAY_CLUSTER_SERVICE_NAME                       = "Gateway_Cluster_Service_Name"
 	NODE_ID                                            = "ReceptorControllerNodeId"
+	PROM_PUSH_GW                                       = "PromPushGW"
```

Comment:
nit-picky:  Can you make this option name a bit more verbose?  Maybe "Prometheus_Push_Gateway"?